thread/common/omrthreadinspect.c

```diff
@@ -332,23 +332,23 @@ omrthread_get_stack_range(omrthread_t thread, void **stackStart, void **stackEnd
 		thread->os_errno = rc;
 		return (J9THREAD_ERR_GETSTACK | J9THREAD_ERR_OS_ERRNO_SET);
 	}
-#endif /* #if (_POSIX_C_SOURCE >= 200112L || _XOPEN_SOURCE >= 600) */
+#endif /* (_POSIX_C_SOURCE >= 200112L || _XOPEN_SOURCE >= 600) */
 	pthread_attr_destroy(&attr);
 
 	/* On Linux, native stack grows from high to low memory */
 	*stackEnd = (void *)((uintptr_t)*stackStart + stackSize);
 	return J9THREAD_SUCCESS;
-#elif defined(OSX)
+#elif defined(OSX) /* defined(LINUX) || defined(AIXPPC) */
 	OSTHREAD osTid = thread->handle;
 	size_t stackSize = 0;
 
 	*stackStart = pthread_get_stackaddr_np(osTid);
 	stackSize = pthread_get_stacksize_np(osTid);
 	*stackEnd = (void *)((uintptr_t)*stackStart + stackSize);
 	return J9THREAD_SUCCESS;
-#else /* defined(OSX) */
+#else /* defined(LINUX) || defined(AIXPPC) */
```

Comment:
as per https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#comment-conditional-compilation-directives-if-ifdef-ifndef-else-elif-endif, the existing `defined(OSX)` comment is correct:
```suggestion
#else /* defined(OSX) */
```

Reply:
done